### PR TITLE
Biosample calc prop to add tissue and organ info for use in faceting

### PR DESCRIPTION
**Pull request by aschroed**
- Added calculated property that adds both tissue source and organ/system info to biosample.
- removed tissue aggregation
- added necessary embeddings
- added facets
- added tests

### CODE REVIEW

File: src/encoded/types/biosample.py [hunk 1]
```diff
@@ -119,6 +119,89 @@ def treatments_summary(self, request, treatments=None):
...
+        if tissue_term:
+            tissue = tissue_term.get('display_title')
+            if 'slim_terms' in tissue_term:
+                slim_terms = self._get_item_info(request, tissue_term.get('slim_terms'), 'ontology_terms')
+                for st in slim_terms:
+                    if st.get('is_slim_for') in ['developmental', 'system', 'organ']:
+                        organ_system.append(st.get('display_title'))
+        return tissue, organ_system
+
+    def _get_item_info(self, request, item, itype):
```
Comment (j1z0) [inline src/encoded/types/biosample.py:135]: Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6380012)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6380012&bid=20695519)

File: src/encoded/tests/test_types_biosample.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import pytest
-
+from unittest import mock
```
Comment (j1z0) [inline src/encoded/tests/test_types_biosample.py:2]: Codacy found an issue: [Unused mock imported from unittest](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6380012)

File: src/encoded/tests/test_types_biosample.py [hunk 3]
```diff
@@ -64,12 +64,12 @@ def test_update_biosample_relation(testapp, human_biosample, biosample_1):
 def test_biosample_calculated_properties(testapp, biosample_1, ):
     """
     Test to ensure the calculated properties are in result returned from testapp
+    These have string 'None' returned if no value as they are used in Item page view
     """
     res = testapp.get(biosample_1['@id']).json
     assert 'modifications_summary' in res
     assert 'modifications_summary_short' in res
     assert 'treatments_summary' in res
-    assert 'biosource_summary' in res
```
Comment (sbreiff) [inline src/encoded/tests/test_types_biosample.py:72]: Why is this line deleted? Is the prop gone or is the behavior different?

Comment (aschroed) [reply src/encoded/tests/test_types_biosample.py]: An oversight - I was going to wrap that calcprop up with all others that used biosource or cell culture details info but Will recommended against it and I forgot to add this back here.

**Review by sbreiff — APPROVED** [review]

File: src/encoded/types/biosample.py [hunk 4]
```diff
@@ -119,6 +119,66 @@ def treatments_summary(self, request, treatments=None):
...
+        "type": "object",
+        "properties": {
+            "tissue_source": {
+                "type": "string"
+            },
+            "organ_system": {
+                "type": "array",
+                "items": {
+                    "type": "string"
+                }
```
Comment (j1z0) [inline src/encoded/types/biosample.py:134]: Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6380012)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6380012&bid=20695519)
